api/AuthenticatorAssertionResponse.json

```diff
@@ -56,7 +56,7 @@
             "version_added": null
           },
           "webview_android": {
-            "version_added": "70"
+            "version_added": false
```

Comment:
This contradicts "This does not replace any values set to a version number" from the description. Can you split out all changes which aren't changes from null to true or false from this PR? That will make it easier to find the evidence for the non-mechanical changes after the fact, and make the final review pass of this PR easier.

Reply:
Whoops, thought that I had updated that part of the description (I originally wrote it not thinking that I would be setting values to false).  This one was set to false from the list of unimplemented interfaces in Webview.  I can give a shot at cherry picking those commits!

Reply:
@jpmedley Thoughts?  This is a change regarding the unexposed APIs in Webview from that list you gave me.

Reply:
I'm not questioning the edits, just requesting that the manual changes are separated from the massive script-made changes.

Reply:
Ah, I see — to be honest, there was a lot of manual edits to the data after the first automated passes that would fix such data, and the Webview changes like this one are a part of it.  It’d be challenging to cherrypick it, and I think it’s more appropriate to leave it together.

Although, since this is a big PR, it may be better to split it similarly to how I did the consistency linting!